addons/test_mail/tests/test_performance.py

```diff
@@ -260,7 +260,7 @@ def test_complex_message_post_template(self):
         record = self.umbrella.sudo(self.env.user)
         template_id = self.env.ref('test_mail.mail_test_tpl').id
 
-        with self.assertQueryCount(admin=141, emp=187):  # com runbot 139 - 185 // test_mail only: 137 - 183
+        with self.assertQueryCount(admin=143, emp=190):  # com runbot 139 - 185 // test_mail only: 137 - 183
```

Comment:
Glad to have added tests so that we spot that kind of issues :) . When performing code-based composer instanciation, values used only for display should not be computed. I suppose added queries come from the follower char field that we don't need outside the form view. Please fix it so that we don't add queries when performing a post with template in code. Thanks !

Reply:
@tde-banana-odoo 

I try with compute field without depends but it does not work on a wizard.so another way is defining a context in the action. what is your opinion about this?